common/models/workspace.js

```diff
@@ -169,6 +178,17 @@ module.exports = function(Workspace) {
         return cb(err);
       }
 
+      var defaultDependencies = template.package.dependencies;
+      if (loopbackVersion === '2.x') {
+        template.package.dependencies =
+          extend(defaultDependencies, DEPENDENCY_2_X);
+      } else if (loopbackVersion === '3.x') {
+        template.package.dependencies =
+          extend(defaultDependencies, DEPENDENCY_3_X);
```

Comment:
Can we reduce duplication in these two branches? E.g.

```js
var loopbackDependencies = loopbackVersion === '2.x' ? DEPENDENCY_2_X : DEPENDENCY_3_X;
template.package.dependencies = extend(defaultDependencies, loopbackDependencies);
```

Also as I was writing this, it occurred to me that these two constants should use plural in their names, because they can contain more than one dependency. I mean `DEPENDENCIES_2_X` and `DEPENDENCIES_3_X`.